test/packetimpact/testbench/connections.go

```diff
@@ -36,6 +36,8 @@ var remoteIPv4 = flag.String("remote_ipv4", "", "remote IPv4 address for test pa
 var localMAC = flag.String("local_mac", "", "local mac address for test packets")
 var remoteMAC = flag.String("remote_mac", "", "remote mac address for test packets")
 
+var SetSameLocalRemoteIP = 0
```

Comment:
We should not be adding global variables.  See below for a different way that we can accomplish this.

Reply:
[GES] Ok will take care.